build-parent/pom.xml

```diff
@@ -550,32 +550,29 @@
                     <artifactId>docker-maven-plugin</artifactId>
                     <version>${version.fabric8.docker.maven.plugin}</version>
                     <extensions>true</extensions>
-                    <executions>
-                        <execution>
-                            <id>default-build</id>
```

Comment:
I: If you hide whitespace changes this becomes more clear, but I essentially just moved the config out of the specific configuration to the main plugin config so that we don't have to duplicate the config for the `default-push` execution.